cmd/kubectl-testkube/commands/generate.go

```diff
@@ -0,0 +1,36 @@
+package commands
+
+import (
+	"github.com/kubeshop/testkube/cmd/kubectl-testkube/commands/common"
+	"github.com/kubeshop/testkube/cmd/kubectl-testkube/commands/crds"
+	"github.com/kubeshop/testkube/cmd/kubectl-testkube/commands/generate"
+	"github.com/kubeshop/testkube/pkg/ui"
+	"github.com/spf13/cobra"
+)
+
+func NewGenerateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "generate <resourceName>",
+		Aliases: []string{},
+		Short:   "Generate resources commands",
+		Run: func(cmd *cobra.Command, args []string) {
+			ui.Logo()
+			cmd.Help()
+		},
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
```

Comment:
Shouldn't we create a library method from this one? we use it everywhere

Reply:
yeah, good idea

Reply:
@vsukhin done